src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -153,6 +153,10 @@
 {
     private static final Logger LOGGER = LogManager.getLogger();
 
+    // De-inlined to ensure
```

Comment:
Ensure what?

Reply:
Oops. "To ensure the two patches don't get out of sync" :P